resources/options_lookup.tsv

```diff
@@ -545,11 +545,11 @@ Insulation Wall	"Steel Stud, R-13"	BuildResidentialHPXML	wall_type=SteelFrame	wa
 Insulation Wall	"Steel Stud, R-25, Grade 3"	BuildResidentialHPXML	wall_type=SteelFrame	wall_assembly_r=11.0	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
 Insulation Wall	"CMU, 6-in Concrete Filled"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=4.3	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
 Insulation Wall	"CMU, 6-in Concrete-Filled, R-10"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=12.0	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
-Insulation Wall	"CMU, 6-in Hollow, Uninsulated"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=0	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
-Insulation Wall	"CMU, 6-in Hollow, R-7"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=0	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
-Insulation Wall	"CMU, 6-in Hollow, R-11"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=0	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
-Insulation Wall	"CMU, 6-in Hollow, R-15"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=0	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
-Insulation Wall	"CMU, 6-in Hollow, R-19"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=0	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
+Insulation Wall	"CMU, 6-in Hollow, Uninsulated"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=2.1	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
+Insulation Wall	"CMU, 6-in Hollow, R-7"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=2.1	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
+Insulation Wall	"CMU, 6-in Hollow, R-11"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=2.1	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
+Insulation Wall	"CMU, 6-in Hollow, R-15"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=2.1	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						
+Insulation Wall	"CMU, 6-in Hollow, R-19"	BuildResidentialHPXML	wall_type=ConcreteMasonryUnit	wall_assembly_r=2.1	wall_color=auto	wall_solar_absorptance=0.7	wall_emittance=0.92																						

```

Comment:
@joseph-robertson Uhh, why do these all have the same assembly R-value? That's not right.

Reply:
A lot of these argument values are still just placeholders.